Source/Microphysics/LagrangianMicrophysics.H

```diff
@@ -0,0 +1,141 @@
+/*! @file LagrangianMicrophysics.H
+ *  \brief Containes the Lagrangian microphysics class
+ */
+
+#ifndef LAGRANGIANMICROPHYSICS_H
+#define LAGRANGIANMICROPHYSICS_H
+
+#ifdef ERF_USE_PARTICLES
+
+#include <utility>
+#include <string>
+
+#include "NullMoistLagrangian.H"
+#include "Microphysics.H"
+
+/* forward declaration */
+class ERFPC;
```

Comment:
@asalmgren @AMLattanzi Instead of forward declaring, should I define `LagrangianMicrophysics` as a templated class with the particle container type as the template parameter? Is that better coding?